roles/network_plugin/cilium/defaults/main.yml

```diff
@@ -145,9 +145,25 @@ cilium_hubble_install: false
 ### Enable auto generate certs if cilium_hubble_install: true
 cilium_hubble_tls_generate: false
 
-# IP address management mode for v1.9+.
-# https://docs.cilium.io/en/v1.9/concepts/networking/ipam/
-cilium_ipam_mode: kubernetes
+# The default IP address management mode is "Cluster Scope".
+# https://docs.cilium.io/en/stable/concepts/networking/ipam/
+cilium_ipam_mode: cluster-pool
+
+# Cluster Pod CIDRs use the kube_pods_subnet value by default.
+# If your node network is in the same range you will lose connectivity to other nodes
+cilium_pool_cidr: "{{ kube_pods_subnet | 10.233.64.0/18 }}"
```

Comment:
@dcwbq  Should this not be `"{{ kube_pods_subnet | default('10.233.64.0/18') }}"` ?